portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -73,13 +73,65 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be required"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeRequired {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132192 CanBeRequired pending implementation
+		ProductMenu.gotoPortlet(
+                        category = "Content &amp; Data",
+                        portlet = "Forms");
+
+                FormsAdminNavigator.gotoAddForm();
+
+                Form.editName(formName = "Form 1");
+
+                Form.gotoAddField(
+                        fieldPositionNumber = "0",
+                        fieldType = "Single Selection",
+                        formPageNumber = "0");
+
+		var fieldSingleSelectionField = FormFields.getFieldName();
+
+                FormFields.enableSwitcher(fieldName = "Required Field");
+
+                Form.save();
+
+                Form.publishForm();
+
+                FormsAdminNavigator.gotoPublishCopyURLLink();
+
+                var publishedFormPublicURL = Form.getPublishedFormPublicURL(
+                        publishedFormPublicURL = "${publishedFormPublicURL}");
+
+                Navigator.openSpecificURL(url = "${publishedFormPublicURL}");
+
+		FormViewBuilder.validateFieldLabelRequired(fieldLabel = "Single Selection");
+
+                Click(
+                        key_text = "Submit",
+                        locator1 = "Button#ANY");
+
+                FormFields.viewFieldValidationErrorMessage(
+                        fieldName = "SingleSelection",
+                        fieldValidationErrorMessage = "This field is required.");
```

Comment:
You can use a macro that is already available which submits the form and expects a validation error message, that is called `FormPortlet.submitWithValidationError`.
You might send the error message as a parameter, so you'll need something like `FormPortlet.submitWithValidationError(validationErrorMessage = "This field is required.")`.

But as another option, you can also just submit the Form and expect an error and keep looking for the validation error on Single Selection field, which would result in a code like this:

```
FormPortlet.submitWithValidationError();

FormFields.viewFieldValidationErrorMessage(                        
        fieldName = "SingleSelection",                        
        fieldValidationErrorMessage = "This field is required.");    
```         

They're both good options and it's up to you to decide :)